docs/upgrade.md

```diff
@@ -199,6 +199,17 @@ Docker images and Debian packages need nothing specific as they already
 include or specify ICU as an explicit dependency.
 
 
+## User directory rebuild
+
+Synapse 1.74 queues a background update
+[to rebuild the user directory](https://github.com/matrix-org/synapse/pull/14643),
```

Comment:
Should we also mention the room directory?

Reply:
Is that what https://github.com/matrix-org/synapse/pull/14643/files#diff-e237abccd3f2c9fd60ccc0c6d1bc1579de88fbed7115c4f637a153286cf92f09R25-R26 does?

Reply:
Oh thats the stats tables, I'm not 100% sure what those are used for tbh. Although the room stats one is used for space hierarchy.